auth/template.yaml

```diff
@@ -1980,6 +1980,7 @@ Resources:
                   - cognito-idp:AdminUserGlobalSignOut
                   - cognito-idp:AdminDeleteUser
                   - cognito-idp:AdminUpdateUserAttributes
+                  - cognito-idp:AdminGetUser
```

Comment:
Probably needs a unit test [here](https://github.com/alphagov/govuk-mobile-backend/blob/2419b5ed7e83b0e0bbb63570c1b6d153cab4547c/auth/tests/unit/shared-signals.unit.test.ts#L92) to check for this addition, cheers